CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+First of all, Welcome, and Thank You!
+
+Graphite has been a community project for years, and it only gets better by the efforts of people who volunteer.  We sincerely appreciate that you care enough to see a problem, take the time to make this project better.  Whether it's in the form of a bug report, code fix, or documentation improvement, we - the Graphite team - appreciate your help.
+
+This is a collection of practices that will make it easier for us to accept your contributions.
+
+
+### Issues
+
+While not mandatory, opening an Issue against the appropriate repo is seldom a bad thing.  We prefer that you have searched through the existing issues.  It's entirely possible that the issue has already been reported.  It may have already been solved.
```

Comment:
"However, we would love if you could take a moment to search through the existing issues first. It's entirely possible that the issue has already been reported, or even better, fixed and resolved."